### PR TITLE
Adding release note on logging forward 4.3 changes 

### PR DESCRIPTION
**Pull request by mburke5678**
Changes required due to https://github.com/openshift/openshift-docs/pull/17825 and https://github.com/openshift/cluster-logging-operator/pull/322

See related addition to docs: https://github.com/openshift/openshift-docs/pull/18906

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge @jcantrill PTAL at https://github.com/openshift/openshift-docs/pull/18908/files#diff-b3bd9c484f4073b02cf7068251b3af28R327-R329.
Do we want to mention in release notes that the user no longer needs to use the fluentd DS to forward logs to an external ES instance?

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -314,6 +314,20 @@ v1.13.
 scaffolded to use Go modules. The `operator-sdk new` command's `--dep-manager`
 flag has likewise been removed.
 
+[discrete]
+[id="ocp-4-3-cluster-logging-changes"]
+==== Cluster logging *out_forward* configuration changes
+
+Changes introduced by the new log forward feature modified the support for *out_forward* starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure out_forward. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed. You can copy your current `secure-forward.conf` section before upgrading to use when creating the `secure-forward` ConfigMap.
```
Comment (jcantrill) [inline release_notes/ocp-4-3-release-notes.adoc]: This is a little confusing.  We need to provide some sort of clarification distinguishing the `secure-forward.conf` section of the 4.2 and prior CM and  the new configmap.  It just needs to be more obvious and explicit

Comment (mburke5678) [reply release_notes/ocp-4-3-release-notes.adoc]: 8020ee4d493205d36603b79f775a289f43994a79

**Comment by mburke5678** [comment]
@jcantrill 
> Please remind me, we edited the DS to update ENV vars for the configuration (e.g. host/port)?  
That is correct. 
https://docs.openshift.com/container-platform/4.2/logging/config/cluster-logging-external.html#cluster-logging-external-syslog_cluster-logging-external

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -341,9 +341,32 @@ v1.13.
...
+In 4.3, the legacy *out_forward* method is deprecated and will be removed in a future release.
+
+When you update to {product-title} 4.3, any existing modifications to the `secure-forward.conf` section of the `fluentd` ConfigMap are removed. You can copy your current `secure-forward.conf` section before updating to use when creating the `secure-forward` ConfigMap object. 
+
 [id="ocp-4-3-unsupported-features"]
 === Unsupported features
 
+[discrete]
+[id="ocp-4-3-cluster-logging-es-external"]
+==== Cluster logging no longer allows forwarding logs using the Fluentd Daemonset
```
Comment (jcantrill) [inline release_notes/ocp-4-3-release-notes.adoc]: by 'editing the Fluentd Daemonset'

File: release_notes/ocp-4-3-release-notes.adoc [hunk 3]
```diff
@@ -341,9 +341,32 @@ v1.13.
...
+When you update to {product-title} 4.3, any existing modifications to the `secure-forward.conf` section of the `fluentd` ConfigMap are removed. You can copy your current `secure-forward.conf` section before updating to use when creating the `secure-forward` ConfigMap object. 
+
 [id="ocp-4-3-unsupported-features"]
 === Unsupported features
 
+[discrete]
+[id="ocp-4-3-cluster-logging-es-external"]
+==== Cluster logging no longer allows forwarding logs using the Fluentd Daemonset 
+
+Due to changes introduced by the new log forwarding feature, you can no longer forward logs to an external Elasticsearch instance using the Fluentd DaemonSet.
```
Comment (jcantrill) [inline release_notes/ocp-4-3-release-notes.adoc]: editing

**Review by jcantrill — COMMENTED** [review]
otherwise LGTM

**Comment by mburke5678** [comment]
@sichvoge PTAL

**Comment by sichvoge** [comment]
LGTM

**Comment by mburke5678** [comment]
@sichvoge Can you take a look? We would like to close on 4.3 docs on Thursday, Brisbane time.

File: release_notes/ocp-4-3-release-notes.adoc [hunk 4]
```diff
@@ -341,9 +341,32 @@ v1.13.
...
 flag has likewise been removed.
 
+[discrete]
+[id="ocp-4-3-cluster-logging-changes"]
+==== Cluster logging *out_forward* configuration changes
+
+Changes introduced by the new xref:../logging/config/cluster-logging-log-forwarding.adoc#cluster-logging-log-forwarding[log forwarding] feature modified the support for the Fluentd *out_forward* plug-in starting with the {product-title} 4.3 release. For the 4.3 release, you can still use the xref:../logging/config/cluster-logging-external.adoc#cluster-logging-fluentd-external_cluster-logging-external[legacy `out_forward` method], without using the new log forwarding feature, which is in Technology Preview. 
+
+To use the legacy *out_forward*, you must create a ConfigMap object to configure *out_forward* instead of editing the `secure-forward.conf` section in the `fluentd` ConfigMap. 
+Additionally, you can add any certificates required by your configuration to a secret that is mounted to the Fluentd pods. See xref:../logging/config/cluster-logging-external.adoc#cluster-logging-fluentd-external_cluster-logging-external[Sending logs to external devices using Fluentd Forward plug-ins].
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: Pods

File: release_notes/ocp-4-3-release-notes.adoc [hunk 5]
```diff
@@ -341,9 +341,32 @@ v1.13.
...
 
+[discrete]
+[id="ocp-4-3-cluster-logging-es-external"]
+==== Cluster logging no longer allows forwarding logs by editing the Fluentd Daemonset 
+
+Due to changes introduced by the new log forwarding feature, you can no longer forward logs to an external Elasticsearch instance by editing the Fluentd DaemonSet.
+
+In previous versions, you could use the `ES_HOST` and `OPS_HOST` environment variables or configure the `fluent-plugin-remote-syslog` plug-in through the `fluentd` Daemonset. 
+
+You can forward logs to an external Elasticsearch instance and other endpoints using the new xref:../logging/config/cluster-logging-log-forwarding.adoc#cluster-logging-log-forwarding[log forwarding] feature or the xref:../logging/config/cluster-logging-external.adoc#cluster-logging-external[Fluentd `forward` plug-ins]. The documentation has been updated to reflect these changes.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-3-release-notes.adoc]: s/has been/is now

**Comment by ahardin-rh** [comment]
@mburke5678 Just some minor suggestions. Otherwise, looks great!

**Pull request merged**